src/api/config/routes/api.rb

```diff
@@ -147,6 +147,14 @@
   resources :request, only: %i[index show create update destroy]
 
   post 'request/:id' => 'request#request_command', constraints: cons
+  controller :request do
+    constraints(cons) do
+      post 'request/:id' => :request_command_diff, constraints: ->(req) { req.params[:cmd] == 'diff' }
```

Comment:
I'm testing this using the apidocs and I don't see the `diff` command, am I doing something wrong?
<img width="1014" height="839" alt="image" src="https://github.com/user-attachments/assets/32523d97-f903-4779-8643-cd8dc1389ba7" />

Reply:
The diff command is documented in a separate entry because its output differs significantly from the other commands. You can find its documentation here: https://api.opensuse.org/apidocs/index#/Requests/post_request__id__cmd_diff

Reply:
Not in my development environment. I've tried `curl -i -u Admin:opensuse -H 'Content-Type: application/octet-stream' -H 'Accept: application/xml' -X POST 'http://localhost:3000/request/1234234?cmd=diff'` and it returns an XML with a 404 error.

What have you tried?

Reply:
> curl -i -u Admin:opensuse -H 'Content-Type: application/octet-stream' -H 'Accept: application/xml' -X POST 'http://localhost:3000/request/1234234?cmd=diff'

I couldn't reproduce it again. I'll come back if I get it again.